test/functional/data/README.md

```diff
@@ -0,0 +1,42 @@
+# Various test vectors
+
+## mainnet_alt.json
+
+For easier testing the difficulty is maximally increased in the first (and only)
+retarget period, by producing blocks approximately 2 minutes apart.
+
+The alternate mainnet chain was generated as follows:
+- use faketime to set node clock to 2 minutes after genesis block
+- mine a block using a CPU miner such as https://github.com/pooler/cpuminer
+- restart node with a faketime 2 minutes later
+
+```sh
+for i in {1..2015}
+do
+ faketime "`date -d @"$(( 1231006505 + $i * 120 ))"  +'%Y-%m-%d %H:%M:%S'`" \
+ bitcoind -connect=0 -nocheckpoints -stopatheight=$i
```

Comment:
This documentation is incomplete, as following the instructions one would run into the check for connections performed by `getblocktemplate`:
https://github.com/bitcoin/bitcoin/blob/45719390a1434ad7377a5ed05dcd73028130cf2d/src/rpc/mining.cpp#L764-L773

To actually follow these instructions on mainnet it is necessary to comment out these lines and recompile.

Reply:
Indeed I left out a few details because it would get tedious.

Reply:
I ended up wasting my own time in #33446 forgetting this, so added more instructions :-)